projects/logistics.md

```diff
@@ -5,6 +5,13 @@ Your Startup Project
 
 Students collaborate on an ambitious software project, a technology startup, from start to finish during the semester.
 
+### Submitting Deliverables
+
+The following schedule lists a few intermediate deliverables.  Add them to your project page and submit a pull request before Friday the week they are due.  Each deliverable is worth 1 point towards your final grade.
+
```

Comment:
We (i) have been telling students to start writing these in some project management tool. There are quite a few that already did. Otherwise they will have to do the work twice and have no place to put future work ideas. I'll merge and amend this? Or you can update.